expectMessage/utils/colorizeLog.ts

```diff
@@ -46,12 +47,42 @@ const createStyle = (cat: any, styleName?: string): string => {
   return styleName ? `\x1b[${styles[cat][styleName.toLowerCase()]}m` : "";
 };
 
-export const colorize = (string: string, customStyle: TerminalCustomStyle = {}): string => {
+const createDefault = (
+  string: string,
+  fg: string,
+  bg: string,
+  effects = ""
+): string => {
   return [
-    createStyle("fg", customStyle.fg),
-    createStyle("bg", customStyle.bg),
-    createStyle("effects", customStyle.effects),
+    createStyle("fg", fg),
+    createStyle("bg", bg),
+    createStyle("effects", effects),
     string.toString().replace(/\s*$/, ""),
-    customStyle.fg || customStyle.bg || customStyle.effects ? createStyle("effects", "reset") : "",
+    fg || bg || effects ? createStyle("effects", "reset") : "",
```

Comment:
This part is really confusing. Could you explain it to me ?

Reply:
I'll leave @Meyclem do that! 
From what I remember, if all are undefined, a reset is necessary to not break the terminal interface